addons/crm/tests/test_crm_lead.py

```diff
@@ -663,3 +663,39 @@ def test_phone_mobile_update(self):
         self.assertEqual(lead.phone, self.test_phone_data[1])
         self.assertEqual(lead.mobile, self.test_phone_data[2])
         self.assertFalse(lead.phone_sanitized)
+
+    @users('user_sales_manager')
+    def test_crm_lead_creation_partner_with_inactive_language(self):
```

Comment:
``test_message_get_suggested_recipients`` maybe ? as currently it is linked only to that method

Please also add it alphabetically. Tests are executed alphabatically so we try to keep their definition using the same ordering.